viewer/add-heading-anchors.js

```diff
@@ -82,13 +82,24 @@ var addHeadingAnchors = {
     });
   },
 
-  registerClipboardHandler: function () {
+  registerClipboardHandlers: function () {
```

Comment:
actual changes here  ( i thought it would be worse, hence all these micro-PRs before the previous ones are reviewed )